### PR TITLE
Add a self-built canary cAdvisor.

### PR DESCRIPTION
**Pull request by vmarmol**
Much credit to @afein 

/cc @rjnagal

### CODE REVIEW

**Comment by vmarmol** [comment]
The goal is to have an Automated Build with an up-to-date cAdvisor. This is not the official release since it is ~600MB in size, but it'll allow people to test the latest features.

**Comment by vishh** [comment]
Nice idea! :+1: LGTM

**Comment by vmarmol** [comment]
It was @afein's idea :D

File: deploy/canary/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM golang:latest
```
Comment (rjnagal) [inline deploy/canary/Dockerfile]: why did we change the tag to beta and have the dir deploy/canary?

**Comment by rjnagal** [comment]
small naming nit, LGTM

File: deploy/canary/Dockerfile (hunk 1 above)
Comment (vmarmol) [inline deploy/canary/Dockerfile]: Moved canary -> beta for builds we make manually. The automated build will be the canary. The thought is that users can try a fixed version without us having to make a release. We can probably replace a lot of the "try vmarmol/cadvisor" with this.

**Pull request merged**